Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+#!groovy
+
+properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '14', numToKeepStr: '10')), [$class: 'ScannerJobProperty', doNotScan: false], [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false], pipelineTriggers([])])
+
+
+node('debian') {
+  try {
+    stage("Plan") {
+      checkout scm
+      sh "git submodule update --init --remote"
```

Comment:
this repo doesn't have any submodules, does it?

Reply:
Thanks, it does not.